src/views/signin-view.js

```diff
@@ -15,119 +17,88 @@ class SignInView extends connect(store)(BaseView) {
     };
   }
 
+  static get styles() {
+    return [customStyles];
+  }
+
   constructor() {
     super();
     this.email = "";
     this.password = "";
     this.name = "";
+
+    this.nameChange.bind(this);
+    this.emailChange.bind(this);
+    this.passwordChange.bind(this);
   }
 
   stateChanged(state) {
     this.error = state.login.loginError;
-    if (state.login.user) {
-      window.location.href = "http://localhost:8080/blogs";
-    }
   }
+
   nameChange(e) {
     this.name = e.target.value;
   }
+
   emailChange(e) {
     this.email = e.target.value;
   }
+
```

Comment:
These 3 methods can be combined in single function if you use object to store name, email and password. Since they are similar variables, grouping them in an object makes sense too.